<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>doctorreg</class>
 <widget class="QDialog" name="doctorreg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1018</width>
    <height>583</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_regis">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>371</width>
     <height>561</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_sumit">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>430</y>
     <width>191</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 255, 255);</string>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>30</y>
     <width>561</width>
     <height>361</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_fill">
      <property name="text">
       <string>Please fill your details</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_name">
        <property name="text">
         <string>Full Name</string>
        </property>
        <property name="buddy">
         <cstring>lineEdit_name</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_name"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_phone">
        <property name="text">
         <string>Phone Number</string>
        </property>
        <property name="buddy">
         <cstring>lineEdit_phone</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_phone"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_gender">
        <property name="text">
         <string>Gender</string>
        </property>
        <property name="buddy">
         <cstring>comboBox_gender</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="comboBox_gender"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_email">
        <property name="text">
         <string>Email Id</string>
        </property>
        <property name="buddy">
         <cstring>lineEdit_email</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEdit_email"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_state">
        <property name="text">
         <string>State</string>
        </property>
        <property name="buddy">
         <cstring>lineEdit_state</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="lineEdit_state"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_city">
        <property name="text">
         <string>City</string>
        </property>
        <property name="buddy">
         <cstring>lineEdit_city</cstring>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="lineEdit_city"/>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_aadhar">
        <property name="text">
         <string>Aadhar Card Number</string>
        </property>
        <property name="buddy">
         <cstring>lineEdit_aadhar</cstring>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="lineEdit_aadhar"/>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_mjrdis">
        <property name="text">
         <string>Allergies/Major Diseases</string>
        </property>
        <property name="buddy">
         <cstring>lineEdit_mjrdis</cstring>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLineEdit" name="lineEdit_mjrdis"/>
      </item>
      <item row="8" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
